app/src/main/java/de/danoeh/antennapod/adapter/CoverLoader.java

```diff
@@ -111,60 +128,52 @@ static class CoverTarget extends CustomViewTarget<ImageView, PaletteBitmap> {
         private final WeakReference<TextView> placeholder;
         private final WeakReference<ImageView> cover;
         private boolean textAndImageCombined;
+        private final WeakReference<TextView> tvFallbackTitle;
 
-        public CoverTarget(TextView txtvPlaceholder, ImageView imgvCover, boolean textAndImageCombined) {
+        public CoverTarget(TextView txtvPlaceholder,
+                           TextView tvFallbackTitle,
+                           ImageView imgvCover, boolean textAndImageCombined) {
             super(imgvCover);
-            if (txtvPlaceholder != null) {
-                txtvPlaceholder.setVisibility(View.VISIBLE);
-            }
             placeholder = new WeakReference<>(txtvPlaceholder);
             cover = new WeakReference<>(imgvCover);
             this.textAndImageCombined = textAndImageCombined;
+            this.tvFallbackTitle = new WeakReference<>(tvFallbackTitle);
```

Comment:
There is now `placeholder` and `fallbackTitle`. This is a bit hard to understand. Shouldn't one of them just be `title`?